src/v2/Apps/Conversation/Components/ConfirmArtworkButton.tsx

```diff
@@ -77,7 +84,7 @@ export const ConfirmArtworkButton: React.FC<ConfirmArtworkButtonProps> = props =
       disabled={props.disabled}
       flexGrow={1}
     >
-      Confirm
+      {props.children || "Confirm"}
```

Comment:
I use children to make the text part of the content of the button. As a counter-example, this is a property called `buttonText` [on Eigen](https://github.com/artsy/eigen/blob/main/src/app/Scenes/Inbox/Components/Conversations/InquiryMakeOfferButton.tsx#L140) and makes it more difficult to read when defininig both buttons as part of a conditional like [in here](https://github.com/artsy/force/pull/9772/files#diff-6d5dd60e277443342ce1f269fd86dfb8d81ceaae975b653fc20d65b88b1d3075R62-R80).